docs/reference/cluster-config.txt

```diff
@@ -23,29 +23,53 @@ agent using a custom configuration file:
     determinedai/determined-agent
     --config-file /etc/determined/agent-config.yaml
 
-Each option in the configuration file corresponds directly to an
-environment variable or command-line option. For example, the master
-configuration file might contain
+The ``agent-config.yaml`` file might contain
 
 .. code:: yaml
 
-   db:
-     host: the-db-host
+  master_host: 127.0.0.1
+  master_port: 8080
+
+to configure the address of the Determined master that the agent will attempt to
+connect to.
 
+Each option in the master or agent configuration file can also be specified as
+an environment variable or a command-line option. To configure the behavior of
+the master or agent using environment variables, specify an environment variable
+starting with ``DET_`` followed by the name of the configuration variable.
+Underscores (``_``) should be used to indicate nested options: for example, the
+``scheduler.fit`` master configuration option can be specified via an
+environment variable named ``DET_SCHEDULER_FIT``.
 
-to configure the host where the PostgreSQL database is running. The same
-thing could be done by starting the master with the
-``DET_DB_HOST=the-db-host`` environment variable or ``--db-host
-the-db-host`` command-line option.
+The equivalent of the agent configuration file shown above can be specified by
+setting two environment variables, ``DET_MASTER_HOST`` and ``DET_MASTER_PORT``.
+When starting the agent as a container, environment variables can be specified
+as part of ``docker run``:
 
-The agent configuration file might contain
+.. code::
+
+  docker run \
+    -e DET_MASTER_HOST=127.0.0.1 \
+    -e DET_MASTER_PORT=8080 \
+    determinedai/determined-agent
+
+The equivalent behavior can be achieved using command-line options:
+
+.. code::
+
+  determined-agent run --master-host=127.0.0.1 --master-port=8080
+
+The same behavior applies to master configuration settings as well. For example,
```

Comment:
I agree we should make this more consistent -- to both enable all options to be set via command-line args and env variables, and to reduce any needless inconsistency between the agent and the master.

In the meantime, do you think this statement is true enough or do we need to add caveats to avoid confusion?